Camera2Extensions/app/src/main/java/com/example/android/camera2/extensions/fragments/CameraFragment.kt

```diff
@@ -790,9 +806,11 @@ class CameraFragment : Fragment(), TextureView.SurfaceTextureListener {
     if (!jpegSupportedSizes.isEmpty()) {
```

Comment:
nit: let's change this to `if (jpegSupportedSizes.isNotEmpty()) {`

Reply:
Done